daemon/graphdriver/overlay2/check.go

```diff
@@ -75,5 +80,23 @@ func hasOpaqueCopyUpBug(d string) error {
 		return errors.New("opaque flag erroneously copied up, consider update to kernel 4.8 or later to fix")
 	}
 
+	// rename "d1" to "d2"
+	if err := os.Rename(filepath.Join(td, "merged", "d1"), filepath.Join(td, "merged", "d2")); err != nil {
+		// if rename fail with syscall.EXDEV, the kernel doesn't enable CONFIG_OVERLAY_FS_REDIRECT_DIR
```

Comment:
Please change the comment to
```go
// if rename failed with syscall.EXDEV, the kernel doesn't have CONFIG_OVERLAY_FS_REDIRECT_DIR enabled
```

(as it's not the kernel which enables or disables its features, but someone who compiled this kernel).